browser/lib/markdown.js

```diff
@@ -211,6 +211,54 @@ class Markdown {
       }
     })
 
+    // Mindmap support
+    this.md.use(require('markdown-it-plantuml'), {
```

Comment:
Instead of using `require` for every type of map, can we just `require` once and use it here?
```js
const plantuml = require('markdown-it-plantuml');
this.md.use(plantuml, {...})
this.md.use(plantuml, {...})
this.md.use(plantuml, {...})
```